imports/plugins/included/product-variant/components/products.js

```diff
@@ -66,6 +66,7 @@ class Products extends Component {
         productIds={getIds({ tags: products })}
         canEdit={Reaction.hasPermission("createProduct")}
         products={products}
+        /* loadMoreProducts={this.props.loadProducts(event)} */
```

Comment:
Is it still needed in future code? If not, delete the line

Reply:
Deleted